README.md

```diff
@@ -1 +1,171 @@
-# Portfolio-setup
\ No newline at end of file
+<div align="center">
+
+  <img src="" alt="Techcolony" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio Mobile First Setup</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> This project is the portfolio mobile first setup, it is created using HTML and CSS with Linters also implemented. It has addition feature of responsivity to breaking points on various  mobile devices.
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML5
+2. CSS3
+3. LINTERS
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/html5/">HTML5</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS3</a></li>
+    <li><a href="">LINTERS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- Added index.html file
+- Added style.css file
+- Added responsiveness to the design
+- Implemented linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming Soon
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+Clone the repository to your local machine using git and open with your desired code editor.
+
+### Prerequisites
+
+In order to run this project you need:
+1. A code editor
+2. Web browser of your choice
+3. github account
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd your folder directory
+  git clone git@github.com:Techcolony/Portfolio-setup.git
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+after cloning the project to your local machine, you can use your code editor to explore the files
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Saheed Oyelola (Techcolony) <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```

Comment:
- [ ] [OPTIONAL] please remove this placeholder texts as they are for your understanding only.